Ed-Fi-Exchange-OSS/EdFi-Project-Buzz
# [FIF-345] - Restore API install

When applied this PR will restore the original build-package.ps1 and nuspec file configs to deploy the API as a Node JS driven Windows service.

TO TEST
1. Build all Buzz Packages  `cd .\EdFi-Project-Buzz\; test.ps1;`
2. Uninstall any prior installs. `cd .\EdFi-Project-Buzz\EdFi.Buzz.Installer\eng\Windows\; uninstall.ps1;`
2. Install the latest. `cd .\EdFi-Project-Buzz\EdFi.Buzz.Installer\eng\Windows\; install.ps1;`

EXPECT
The edfi-buzz-api Windows Service should be installed and running.
http://localhost:3000/graphql should GraphQL requests.
